src/classes/groundplaces.ts

```diff
@@ -201,31 +195,119 @@ export class GroundPlacesController {
     intoStopGroupGpuid: StopGroupGpuid,
   ): void {}
 
+  /**
+   * @description Add a stopGroup to a stopCluster.
+   * @param {StopGroupGpuid} stopGroupGpuidToAdd - Ground place unique identifier of the StopGroup to add.
+   * @param {StopClusterGpuid} intoStopClusterGpuid - Ground Place unique identifier of the new StopCluster parent.
+   * @returns {void}
+   */
+  public addStopGroupToStopCluster(stopGroupGpuidToAdd: StopGroupGpuid, intoStopClusterGpuid: StopClusterGpuid): void {
+    // Check if the StopGroup specified exist in the Ground places
+    this.storageService.getStopGroupByGpuid(stopGroupGpuidToAdd);
+
+    const stopClusterParent: StopCluster = this.storageService.getStopClusterByGpuid(intoStopClusterGpuid);
+
+    // Check if the StopGroup does not already exist in the new StopCluster parent
+    const isStopGroupAlreadyExist = Boolean(
+      stopClusterParent.childs.find((stopGroupGpuid: StopGroupGpuid) => stopGroupGpuid === stopGroupGpuidToAdd),
+    );
+
+    if (isStopGroupAlreadyExist) {
+      throw new Error(
+        `The StopGroup with the Gpuid "${stopGroupGpuidToAdd}" cannot be added to the StopCluster with the Gpuid "${intoStopClusterGpuid}" because it already exist in it.`,
+      );
+    }
+
+    stopClusterParent.childs.push(stopGroupGpuidToAdd);
+
+    // Update the place with the new StopCluster parent
+    this.storageService.replacePlace(stopClusterParent);
+  }
+
+  /**
+   * @description Remove a stopGroup from a stopCluster.
+   *
+   * Warning: The StopGroup cannot be without StopCluster parent after this operation.
+   * @param {StopGroupGpuid} stopGroupGpuidToRemove - Ground place unique identifier of the SopGroup to remove.
+   * @param {StopClusterGpuid} stopClusterGpuidParent - Ground place unique identifier of the StopCluster parent.
+   * @returns {void}
+   */
+  public removeStopGroupFromStopCluster(
+    stopGroupGpuidToRemove: StopGroupGpuid,
+    stopClusterGpuidParent: StopClusterGpuid,
+  ): void {
+    const stopClusterParent: StopCluster = this.storageService.getStopClusterByGpuid(stopClusterGpuidParent);
+    const groundPlaces: GroundPlace[] = this.storageService.getGroundPlaces();
+    const cloneGroundPlaces: GroundPlace[] = cloneDeep(this.storageService.getGroundPlaces());
+
+    const stopGroupIndex: number = stopClusterParent.childs.findIndex(
+      (stopGroupGpuid: StopGroupGpuid) => stopGroupGpuid === stopGroupGpuidToRemove,
+    );
+
+    if (stopGroupIndex === -1) {
+      throw new Error(
+        `The StopGroup with the Gpuid "${stopGroupGpuidToRemove}" cannot be removed from the StopCluster with the Gpuid "${stopClusterGpuidParent}" because it does not belong to it.`,
+      );
+    }
+
+    // Remove the reference of the StopGroup Gpuid inside the StopCluster parent
+    stopClusterParent.childs.splice(stopGroupIndex, 1);
```

Comment:
Wrning. 

When you get via this methods. 
`this.storageService.getStopClusterByGpuid(stopClusterGpuidParent);`
This does NOT deep copy the object. But only a ref !

Reply:
Yup! You must not have seen it in the PR, but I've just change the way the **get** methods were used, to return `cloneDeep(place)`, instead of only **place**. Now, we don't have the reference but a new object cloned from the current one retrieved.

I'm talking to Théo about it in this thread https://github.com/tictactrip/ground-places-sdk/pull/5#discussion_r540956656